rfc/README.md

```diff
@@ -63,3 +63,4 @@ The list of all RFCs can be found here.
 | 37 | [Hudi metadata based bloom index] | `UNDER REVIEW` |
 | 38 | [Spark Datasource V2 Integration] | `UNDER REVIEW` |
 | 39 | [Incremental source for Debezium] | `UNDER REVIEW` |
+| 40 | [Hudi Snowflake Integration] | `UNDER REVIEW` |
```

Comment:
this needs to go into its own PR. 40 is already taken on master for eg

Reply:
Sorry, I didn't know, I updated it to 41, is it okay to leave it in this same PR? or do you still want me to separate it out?